docs/reference/query-dsl/geo-distance-query.asciidoc

```diff
@@ -267,7 +267,7 @@ The following are options allowed on the filter:
 
 `distance_type`::
 
-    How to compute the distance. Can either be `arc` (default), or `plane` (faster, but inaccurate on long distances and close to the poles).
+    deprectated[7.1.0] How to compute the distance. This is handled internally
```

Comment:
Same as above. Note the version is incorrect.

Reply:
The deprecated admon is also misspelled. Should be `deprecated:[7.13.0]`.